indicators/indicators.py

```diff
@@ -96,10 +96,7 @@ def convert_times_to_years(time_da):
             cftime.num2date(t / 1e9, "seconds since 1970-01-01")
             for t in time_da.values.astype(int)
         ]
-    elif isinstance(
-        time_da.values[0],
-        cftime._cftime.Datetime360Day,
-    ) or isinstance(
+    elif isinstance(time_da.values[0], cftime._cftime.Datetime360Day,) or isinstance(
```

Comment:
Think we may have some format-bouncing here? Noticed this one going the other way before maybe, just something to look out for

Reply:
Yeah I think this just got re-formatted by `black`, code did not change as far as I can tell

Reply:
We should spend a few minutes on Monday making sure that everyone's IDE is setup to automatically run Python Black on save as that will prevent much of this format bounce going forward.